db/seeds.rb

```diff
@@ -5,7 +5,7 @@
                                                                   .split("\r\n")
                                                                   .delete_if { |line| line.starts_with?('#') }
                                                                   .map { |line| [line] }
-MaliciousUrl.import columns, values, validate: false, on_duplicate_key_ignore: true
+# MaliciousUrl.import columns, values, validate: false, on_duplicate_key_ignore: true
```

Comment:
Do we not want this anymore?

Reply:
Just noticed that locally. Was an unintentional commit.
```suggestion
MaliciousUrl.import columns, values, validate: false, on_duplicate_key_ignore: true
```

Reply:
Commenting this line takes the `rake db:seed` step of rebuilding the app from more than 10-15 minutes to just about 30 seconds.